builtin/algorithm/mod.rs

```diff
@@ -1,31 +1,39 @@
 // Copyright © 2024 The µcad authors <info@ucad.xyz>
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
-use microcad_lang::{objects::*, resolve::*};
+use microcad_lang::{eval::Builtin, objects::*, resolve::*, syntax::Identifier, value::Value};
 
 /// Creates a node containing a difference algorithm
 fn difference() -> Symbol {
-    Symbol::new_builtin_module("difference", &|_, _| Ok(algorithm::difference()))
+    Symbol::new_builtin(Builtin::new(Identifier::no_ref("difference"), &|_, _| {
+        Ok(Value::Node(algorithm::difference()))
+    }))
```

Comment:
Why not like this, and do the complicated part within `new_builtin()`?

```suggestion
    Symbol::new_builtin("difference", &|_, _| {
        Ok(Value::Node(algorithm::difference())
    })
```